### PR TITLE
Add padding to definition list (single contact)

### PR DESCRIPTION
**Pull request by drmenzelit**
Pull Request for Issue #173 .

### Summary of Changes
Added padding left (right on rtl) to definition list

### Testing Instructions
Run npm

### Expected result

grafik

### Actual result
No padding

### CODE REVIEW

File: templates/cassiopeia/scss/blocks/_global.scss [hunk 1]
```diff
@@ -62,6 +62,13 @@ small,
 
 dd {
   margin-bottom: 0;
+  padding: 0 0 0 ($cassiopeia-grid-gutter*2);
+}
+
+[dir="rtl"] dd {
+  padding: 0 ($cassiopeia-grid-gutter*2) 0 0 ;
```
Comment (richard67) [inline templates/cassiopeia/scss/blocks/_global.scss]:
```suggestion
  padding: 0 ($cassiopeia-grid-gutter*2) 0 0;
```

**Comment by richard67** [comment]
@drmenzelit I've allowed myself to commit the CS fix here which I had suggested. If you have to continue work on this PR locally for some reason, pull changes from remote first.

**Comment by chmst** [comment]
Tested successfully. The whole contact form causes several issues which are to be resolved in laoyuts, not cassiopeia.

**Comment by richard67** [comment]
I have tested this PR :white_check_mark: with success.

**Comment by drmenzelit** [comment]
@richard67 thank you... my stylelint apparently missed the wrong space

**Comment by richard67** [comment]
Since I'm not an scss expert I am not the one who should approve this PR.

Maybe @hans2103 can? Or @chmst ?

**Comment by infograf768** [comment]
Just FYI, see https://github.com/joomla/joomla-cms/issues/31104

**Review by chmst — APPROVED** [review]

**Comment by richard67** [comment]
Thanks!

**Pull request merged**